app/assets/javascripts/common/reports.js

```diff
@@ -471,22 +473,21 @@ Reports = function ({ withLtfu, showGoalLines }) {
   function goalPeriodValue(periodValues) {
     const dateKeysArray = Object.keys(periodValues);
     const decemberKeys = dateKeysArray.filter((item) => item.includes("Dec"));
-    
-    const indexOfLatestDecember = dateKeysArray.indexOf(
-      decemberKeys[decemberKeys.length - 1]
-    );
+    const mostRecentDecemberKey = decemberKeys[decemberKeys.length - 1];
+    const indexOfMostRecentDecemberInPeriodValues = dateKeysArray.indexOf(mostRecentDecemberKey);
```

Comment:
The `InPeriodValues` bit is implied, no? `indexOfLatestDecember` seems like a good name by itself :D

Reply:
Reverted to 'indexOfLatestDecember'